app/src/components/tables/BlockTable/BlocksTable.test.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Table } from 'src/components';
+import {
+  getMockBlocks,
+  mockBlocksTableColumns,
+  getMockBlocksTableFooter,
+  getMockBlocksTableHeader,
+} from 'src/mocks/mock-blocks-table';
+import { formatDate, truncateHash } from 'src/utils';
+import { render } from '../../../test-utils';
+
+const blocks = getMockBlocks();
+const mockBlocksTableFooter = getMockBlocksTableFooter();
+const mockBlocksTableHeader = getMockBlocksTableHeader();
+
+describe('BlocksTable', () => {
+  it('should render the BlocksTable', () => {
+    render(
+      <Table
+        tableBodyLoading={false}
+        header={mockBlocksTableHeader}
+        columns={mockBlocksTableColumns}
+        isLastPage
```

Comment:
I realize this is just a test so it probably doesn't matter, but why do we want `isLastPage` to be true?

Reply:
it is a required prop so I included it in there as true

Reply:
Right, but it just seems weird that you'd be testing data on the last page instead of the first (or any others)